doc/source/tune/user-guide.rst

```diff
@@ -1121,10 +1121,6 @@ These are the environment variables Ray Tune currently considers:
   Ctrl+C) to gracefully shutdown and do a final checkpoint. Setting this variable
   to ``1`` will disable signal handling and stop execution right away. Defaults to
   ``0``.
-* **TUNE_FORCE_TRIAL_CLEANUP_S**: By default, Ray Tune will gracefully terminate trials,
-  letting them finish the current training step and any user-defined cleanup. 
-  Setting this variable to a non-zero, positive integer will cause trials to be forcefully
-  terminated after a grace period of that many seconds. Defaults to ``0``.
```

Comment:
Would it be possible to keep this feature in? Ensuring that Tune exits in constant time after reaching the time deadline is useful for eg. benchmarks and competitions, and allows to save cost by not having to waste time on graceful termination, especially if iterations take a long time.

Reply:
Please take a look. I also added a test to test force clean up.